src/components/visit/formField.tsx

```diff
@@ -0,0 +1,42 @@
+interface FormFieldProps {
```

Comment:
Best to stick to the same naming conventions for all your files, this should be `FormField.tsx` rather than `formField.tsx`